### PR TITLE
pkg-config-wrapper 0.2.8 (new formula)

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/pkg-config-wrapper.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class PkgConfigWrapper < Formula
+  desc "Easier way to include C code in your Go program"
+  homepage "https://github.com/influxdata/pkg-config"
+  url "https://github.com/influxdata/pkg-config/archive/v0.2.8.tar.gz"
+  sha256 "9d3f3bbcac7c787f6e8846e70172d06bd4d7394b4bcd0b8572fe2f1d03edc11b"
+  license "MIT"
+  head "https://github.com/influxdata/pkg-config.git", branch: "master"
+
+  depends_on "go" => :build
+  depends_on "pkg-config" => [:build, :test]
```
Comment (SMillerDev) [inline Formula/pkg-config-wrapper.rb:10]: Isn't it needed at runtime to actually use pkg-config though?

Comment (chenrui333) [reply Formula/pkg-config-wrapper.rb:10]: it is a wrapper over pkg-config during the build time, that is why I did not specify it as runtime dependency. let me know what you think.

**Comment by chenrui333** [comment]
go for the merge?

File: Formula/pkg-config-wrapper.rb (hunk 1 above)
Comment (SMillerDev) [reply Formula/pkg-config-wrapper.rb:10]: But it uses pkg-config when you use it. So it's a runtime dependency.

Comment (chenrui333) [reply Formula/pkg-config-wrapper.rb:10]: sounds good, let me move to runtime dependency then.

**Review by branchv — APPROVED** [review]

**Pull request closed without merging**